src/nifgen/system_tests/test_system_nifgen.py

```diff
@@ -23,6 +23,15 @@ def session():
         yield simulated_session
 
 
```

Comment:
Please improve documentation where the locks are created.
Internal NI bug should be referenced.

Reply:
Comment updated